### PR TITLE
add success message and rebase information on create_tags command

### PR DESCRIPTION
**Pull request by Tashima42**
For some reason, the `git rebase` command always writes to STDERR and when logging in Go, the rebase command output deletes everything that comes before in the line.

```diff
- INFO[0232] Successfully rebased and updated detached HEAD. # expected
+ Successfully rebased and updated detached HEAD. # result
```

Closes: #175

### CODE REVIEW

File: release/k3s/k3s.go [hunk 1]
```diff
@@ -195,41 +197,42 @@ func (r *Release) SetupK8sRemotes(_ context.Context, ghClient *github.Client) er
 	return nil
 }
 
-func (r *Release) RebaseAndTag(_ context.Context, ghClient *github.Client) ([]string, error) {
-	if err := r.gitRebaseOnto(); err != nil {
-		return nil, err
+func (r *Release) RebaseAndTag(_ context.Context, ghClient *github.Client) (string, []string, error) {
```
Comment (briandowns) [inline release/k3s/k3s.go:200]: Let's make the rebase output the second value returned here.

File: release/k3s/k3s.go [hunk 2]
```diff
@@ -181,6 +181,8 @@ func (r *Release) SetupK8sRemotes(_ context.Context, ghClient *github.Client) er
 		}
 	}
 
+
```
Comment (briandowns) [inline release/k3s/k3s.go:184]: Is your editor configured to format your files?

File: release/k3s/k3s.go [hunk 3]
```diff
@@ -252,31 +255,31 @@ func getAuth(privateKey string) (ssh.AuthMethod, error) {
...
 	if err := cleanGitRepo(dir); err != nil {
-		return err
+		return "", err
 	}
 	if _, err := runCommand(dir, "rm", "-rf", "_output"); err != nil {
-		return err
+		return "", err
 	}
 
 	commandArgs := strings.Split(fmt.Sprintf("rebase --onto %s %s %s-k3s1~1",
```
Comment (briandowns) [inline release/k3s/k3s.go:269]: Can we add another entry with `2>&1`?

File: release/k3s/k3s.go (hunk 2 above)
Comment (Tashima42) [reply release/k3s/k3s.go]: No, but I tried using lunarvim and it probably did it.

File: release/k3s/k3s.go (hunk 3 above)
Comment (Tashima42) [reply release/k3s/k3s.go:266]: I tried it in a couple of different ways, but it always ends up with an error:
```
FATA[0005] failed to rebase and create tags: exit status 129: usage: git rebase [-i] [options] [--exec <cmd>] [--onto <newbase> | --keep-base] [<upstream> [<branch>]]
```

**Review by brooksn — APPROVED** [review]

File: release/k3s/k3s.go [hunk 4]
```diff
@@ -203,41 +202,42 @@ func (r *Release) SetupK8sRemotes(_ context.Context, ghClient *github.Client) er
 	return nil
 }
 
-func (r *Release) RebaseAndTag(_ context.Context, ghClient *github.Client) ([]string, error) {
-	if err := r.gitRebaseOnto(); err != nil {
-		return nil, err
+func (r *Release) RebaseAndTag(_ context.Context, ghClient *github.Client) ([]string, string, error) {
+	rebaseOut, err := r.gitRebaseOnto()
```
Comment (briandowns) [inline release/k3s/k3s.go:206]: Is this not formatted or the way GitHub is showing it?

Comment (Tashima42) [reply release/k3s/k3s.go:206]: I think it's just Github 
https://github.com/rancher/ecm-distro-tools/blob/b517d52fd65c99c818eacd21d25c9c79f60b0491/release/k3s/k3s.go#L205-L206

**Review by briandowns — APPROVED** [review]

**Pull request merged**